internal/database/autocontact.go

```diff
@@ -0,0 +1,169 @@
+package database
```

Comment:
Why's this in `database`? `onboard` seems like a much nicer place for it.

Reply:
Onboard is probably a bad place for it semantically. I'll put it in control methinks